lib/buildkite/test_collector/minitest_plugin/trace.rb

```diff
@@ -74,7 +74,7 @@ def failure_expanded
         # remove the first line of message from the first failure
         # to avoid duplicate line in Test Analytics UI
         messages = strip_invalid_utf8_chars(failure.message).split("\n")
-        messages = messages[1..] if index.zero?
+        messages = messages[1..-1] if index.zero?
```

Comment:
Is this change just for clarity?

Reply:
Functionally the syntax is equivalent but Ruby endless ranges was only introduced in Ruby 2.6. 

https://rubyreferences.github.io/rubychanges/2.6.html#endless-range-1